opentelemetry-journald-logs/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "opentelemetry-journald-logs"
```

```suggestion
name = "opentelemetry-journald"
```

---

How about `opentelemetry-exporter-journald`?

```suggestion
name = "opentelemetry-exporter-journald"
```

---

I like specific names, so opentelemetry-exporter-journald sounds good